distribution/docs/src/main/resources/IntContents.adoc

```diff
@@ -1,10 +1,245 @@
+----
++
+Where:
+*-H* adds an HTTP header. In this case, Content-type header `application/json;id=geojson` is added to match the data being sent in the request.
+*-i* requests that HTTP headers are displayed in the response.
+*-X* specifies the type of HTTP operation. For this example, it is necessary to POST (ingest) data to the server.
+*-d* specifies the data sent in the POST request. The @ character is necessary to specify that the data is a file.
++
+The last parameter is the URL of the server that will receive the data.
++
+This should return a response similar to the following (the actual catalog ID in the id and Location URL fields will be different):
++
+.Sample Response
+[source,http,linenums]
+----
+HTTP/1.1 201 Created
+Content-Length: 0
+Date: Mon, 22 Apr 2013 22:02:22 GMT
+id: 44dc84da101c4f9d9f751e38d9c4d97b
+Location: https://localhost:8993/services/catalog/44dc84da101c4f9d9f751e38d9c4d97b
+Server: Jetty(7.5.4.v20111024)
+----
++
+. Verify the entry was successfully ingested by entering in a browser the URL returned in the POST response's HTTP header. For instance in our example, it was `/services/catalog/44dc84da101c4f9d9f751e38d9c4d97b`. This should display the catalog entry in XML within the browser.
+. Verify the catalog entry exists by executing a query via the OpenSearch endpoint.
+. Enter the following URL in a browser /services/catalog/query?q=ddf. A single result, in Atom format, should be returned.
+
+==== Use of the Content Framework
+Using the Content framework's directory monitor, ingest a file so that it is stored in the content repository with a metacard created and inserted into the Catalog.
+
+. In the Web Console, select the Configuration tab.
+. Select the *Content Directory Monitor*.
+. Set the directory path to *inbox*.
+. Click the *Save* button.
+. Copy the attached link:geojson_valid.json[geojson] file to the `<DISTRIBUTION_INSTALL_DIR>/inbox` directory.
++
+The Content Framework will:
++
+.. ingest the file,
+.. store it in the content repository at <DISTRIBUTION_INSTALL_DIR>/content/store/<GUID>/geojson_valid.json,
+.. look up the GeoJson Input Transformer based on the mime type of the ingested file,
+.. create a metacard based on the metadata parsed from the ingested GeoJson file, and
+.. insert the metacard into the Catalog using the CatalogFramework.
++
+Note that XML metadata for text searching is not automatically generated from GeoJson fields.
+. Verify GeoJson file was stored using the Content REST endpoint.
+.. Install the feature content-rest-endpoint using the Features tab in the Web Console.
+.. Send a GET command to read the content from the content repository using the Content REST endpoint. This can be done using curl command below. Note that the GUID will be different for each ingest. The GUID can be determined by           going to the <DISTRIBUTION_INSTALL_DIR>/content/store directory and copying the sub-directory in this folder (there should only be one).
+
+.*NIX Example
```

Comment:
Agreed, even though the command might be the same (not sure you even need to specify the .exe) in this case, a precedent was kind of set to show Windows Example... Also, has anyone "tested" these commands recently on *nix and/or Windows?